services/operatorsinfo/operatorsinfo_subgraph_test.go

```diff
@@ -0,0 +1,60 @@
+package operatorsinfo
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shurcooL/graphql"
+	"github.com/stretchr/testify/assert"
+)
+
+type mockGraphQLQuerier struct {
+	QueryFn func(ctx context.Context, q any, variables map[string]any) error
+}
```

Comment:
this reads a bit weird. Is it needed to have the QueryFn as a struct field here? Why not define QueryFn as a method? It's only defined once in the test below, so doesn't seem like this kind of abstraction is needed, and its hard to understand why its being defined like this.

Reply:
hmm I see. Still find it weird but chatgpt convinced me its doable.